test/OpenTelemetry.Tests/Logs/LoggingProviderTests.cs

```diff
@@ -0,0 +1,87 @@
+// <copyright file="LoggingProviderTests.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+#if !NET461
+using System;
+
+using OpenTelemetry.Resources;
+
+using Xunit;
+
+namespace OpenTelemetry.Logs.Tests
+{
+    public sealed class LoggingProviderTests
+    {
+        [Fact]
+        public void VerifyDefaultBehavior()
+        {
+            var options = new OpenTelemetryLoggerOptions();
+            var provider = new OpenTelemetryLoggerProvider(options);
+
+            Assert.Contains(provider.Resource.Attributes, (kvp) => kvp.Key == "service.name" && kvp.Value.ToString() == "unknown_service:testhost");
+        }
+
+        [Fact]
+        public void VerifyResourceBuilderAddService()
+        {
+            var options = new OpenTelemetryLoggerOptions
+            {
+                ResourceBuilder = ResourceBuilder.CreateDefault().AddService(serviceName: "MyService", serviceVersion: "1.2.3"),
+            };
+            var provider = new OpenTelemetryLoggerProvider(options);
```

Comment:
this is not how users would be setting resource, so I'd suggest to keep the unit test doing what user would be doing. (there is no public setter `ResourceBuilder`, tests are able to access internals)

Reply:
its okay to do the test additions separately.

Reply:
if it's okay to contribute tests with this PR I would like to do that to avoid creating MORE PRs :)